app/models/intake.rb

```diff
@@ -253,6 +253,7 @@
 #
 # Foreign Keys
 #
+#  fk_rails_...  (bank_account_id => bank_accounts.id)
```

Comment:
not following this new bank account foreign key...maybe it will become apparent as i keep reading

Reply:
it was just missing... we had an association between intakes and bank accounts but no foreign key to guarantee you can't put 9999 in there or some other arbitrary value

Reply:
on further investigation this column should be dropped... we have columns in the DB for both `bank_accounts.intake_id` and `intakes.bank_account_id` but only the former is populated